Terminal.Gui/Drawing/Aligner.cs

```diff
@@ -108,10 +108,11 @@ public int ContainerSize
             spacesToGive = containerSize - totalItemsSize;
         }
 
+        AlignmentModes mode = alignmentMode & ~AlignmentModes.AddSpaceBetweenItems; // copy to avoid modifying the original
```

Comment:
🙏 

Did you profile this? I have. This makes a surprisingly significant difference (a good one).

Thank you!